model/shared/src/main/scala/hmda/model/fi/FIData.scala

```diff
@@ -0,0 +1,20 @@
+package hmda.model.fi
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.fi.ts.TransmittalSheet
+import scala.scalajs.js.annotation.JSExport
+import scala.scalajs.js.JSConverters._
+
+case class FIData(ts: TransmittalSheet, lars: Iterator[LoanApplicationRegister]) {
+  def toCSV: String = {
```

Comment:
Correct. This may not be needed. I think it's more important to have the individual CSV representation of TransmittalSheet and LAR